tests/render/test_column_section_renderer.py

```diff
@@ -834,7 +844,13 @@ def test_ExpectationSuiteColumnSectionRenderer_expectation_with_single_string_me
                                     },
                                     "subheader": "Notes:",
                                     "text": [
-                                        "This is just a single string, assigned to the 'content' key of a notes dict."
+                                        {
+                                            "content_block_type": "markdown",
+                                            "markdown": "This is just a single string, assigned to the 'content' key of a notes dict.",
```

Comment:
`assigned to the 'content' key of a notes dict` that seems to be no longer correct